webapp/lib/model.g.dart

```diff
@@ -870,6 +875,7 @@ StreamSubscription<List<DocSnapshot>> listenForUpdates<T>(
     void Function(List<T> added, List<T> modified, List<T> removed) listener,
     String collectionRoot,
     T Function(DocSnapshot doc) createModel,
+    [OnErrorListener onErrorListener]
```

Comment:
since we're passing the `onErrorListener` in all calls to `listenForUpdates`, then let's not make it optional.
```suggestion
    OnErrorListener onErrorListener,
```